.changeset/calm-steaks-eat.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/client-js': patch
+---
+
+Added optional plan content fields to PlanResume so hosts can preserve submitted plans in approval history.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)